Jenkinsfile

```
@@ -385,7 +385,7 @@ boolean skip_ftest_hw(String size) {
 
 boolean skip_bandit_check() {
     return cachedCommitPragma(pragma: 'Skip-python-bandit',
-                              def_val: 'true') == 'true' ||
+                              def_val: 'false') == 'true' ||
```

You don't really need the `def_val:` here now:

```suggestion
    return cachedCommitPragma(pragma: 'Skip-python-bandit') == 'true' ||
```

---

I needed to push anyway to get another shot at testing so I've resolved this, but the PR will need re-approving now.